src/main/resources/config/application.yml

```diff
@@ -13,6 +13,28 @@
 # Full reference is available at:
 # http://docs.spring.io/spring-boot/docs/current/reference/html/common-application-properties.html
 # ===================================================================
+zuul:
+    host:
+        max-total-connections: 1000
+        max-per-route-connections: 100
+    semaphore:
+        max-semaphores: 500
+    retryable: true
+    routes:
+        ManagementPortal:
+            path: /oauthserver/oauth/**
+            url: http://localhost:8080/oauth
```

Comment:
For `prod` profile, this needs to be `url: http://localhost:8080/managementportal/oauth` due to the context path being set.